### PR TITLE
Status/2024Q3: Create dhclient changes report

### PR DESCRIPTION
**Pull request by ica574**
Create a folder report-2024-07-2024-09, and list an AsciiDoc report on the changes made to dhclient within 2024Q3. @cperciva

### CODE REVIEW

**Comment by cperciva** [comment]
@lsalvadore Isaac is my GSoC student; can you review his status report and commit it?

File: website/content/en/status/report-2024-07-2024-09/dhclient.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+=== Status Report Sample - This Will Become The Title
```
Comment (pauamma) [inline website/content/en/status/report-2024-07-2024-09/dhclient.adoc:1]: This needs to be the report title, perhaps "=== Changes to dhclient"

File: website/content/en/status/report-2024-07-2024-09/dhclient.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2024Projects/SpeedingUpTheFreeBSDBootProcess[Speeding up the FreeBSD boot process] URL: link:https://wiki.freebsd.org/SummerOfCode2024Projects/SpeedingUpTheFreeBSDBootProcess[] +
+
+link:https://github.com/freebsd/freebsd-src/pull/1368[dhclient Pull Request] URL: link:https://github.com/freebsd/freebsd-src/pull/1368[] +
+
+Contact: Isaac Cilia Attard <icattard@FreeBSD.org>
+
+As part of my Google Summer of Code 2024 project, involving speeding up the FreeBSD boot process, I have worked on decreasing the time it takes for ARP resolution within dhclient to happen. This involved reducing the default ARP resolution timeout from 2000 ms to 250 ms, and adding an option to disable it altogether. The latter is useful within cloud environments, where a node is certain to have an IP address allotted to it.
+
+As a consequence of this, connecting to a DHCP network is now faster, including the boot process during which this happens. The speedup experienced is that of about 2 seconds.
```
Comment (pauamma) [inline website/content/en/status/report-2024-07-2024-09/dhclient.adoc:12]: s/that of //

**Review by pauamma — COMMENTED** [review]
Approved (with status editor hat on) once these changes are made.

**Review by lsalvadore — APPROVED** [review]
Approved once @pauamma's comments are fixed. Once they are, I will commit the report.
Thanks for submission and review!

File: website/content/en/status/report-2024-07-2024-09/dhclient.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
+=== Changes to dhclient to speed up the FreeBSD boot process ===
```
Comment (pauamma) [inline website/content/en/status/report-2024-07-2024-09/dhclient.adoc:1]: `===` at the end isn't needed.

**Review by pauamma — APPROVED** [review]
Fixable on commit.

**Review by pauamma — APPROVED** [review]

**Pull request closed without merging**